src/deephaven_ib/_query_inputs.py

```diff
@@ -0,0 +1,55 @@
+"""Inputs to queries.
+
+Currently Deephaven has an open ticket for using variables in queries from within python modules.
+This module is to house nasty workaround code until the problem has been resolved.
+
+https://github.com/deephaven/deephaven-core/issues/1072
+"""
+
+# TODO: When ticket #1072 is resolved, the following code should be replaced with something cleaner
+# To work around the problem, variables have to be fed into QueryScope.
+# Since python objects can't be inserted into QueryScope, PythonFunctions were created, which are Java objects.
+# PythonFunctions only take a single argument, so the python functions had to be changed to take only one arument.
+
+from typing import Union
+
+import jpy
+from deephaven import PythonFunction
+from deephaven.conversion_utils import NULL_DOUBLE
+
+_QueryScope = jpy.get_type("io.deephaven.engine.table.lang.QueryScope")
+
+
+def __deephaven_ib_float_value(s: str) -> Union[float, None]:
+    try:
+        return float(s)
+    except ValueError:
+        return NULL_DOUBLE
+
```

Comment:
is NaN bettter?

Reply:
Not in this case.  IB decided to encode a column of values as strings.  Most of the values are really floats, but some are really strings (e.g. "Currency").  This function is just used to create a column where the float values are floats, and the other stuff is null.